forge-gui/src/main/java/forge/player/PlayerControllerHuman.java

```diff
@@ -469,12 +469,12 @@ public Integer announceRequirements(final SpellAbility ability, final String ann
                 if (costX != null) {
                     max = Math.min(max, costX);
                 }
-                if (cost.hasManaCost() && !cost.getCostMana().canXbe0()) {
-                    canChooseZero = false;
+                if (cost.hasManaCost()) {
+                    xMin = cost.getCostMana().getXMin();
```

Comment:
I don't think it works in cases like _Corpseweft_ since you'd be overwriting the param?

Reply:
Good catch